build.rs

```diff
@@ -9,7 +9,10 @@ fn main() {
     // get timestamp
     let now = match env::var("SOURCE_DATE_EPOCH") {
         Ok(val) => {
-            let naive = NaiveDateTime::from_timestamp(val.parse::<i64>().unwrap(), 0);
+            let naive = match NaiveDateTime::from_timestamp_opt(val.parse::<i64>().unwrap(), 0) {
```

Comment:
Is there a reason we're doing unwrap()? `unwrap_or()` would be safer?

Reply:
just the way it was ... happy to do otherwise if folks like

Reply:
this is just build.rs it should fail here when it tries to compile but got in invalid SOURCE_DATE_EPOCH

Reply:
Fair enough